src/Codeception/Module/Symfony2.php

```diff
@@ -93,17 +99,18 @@ class Symfony2 extends Framework implements DoctrineProvider, PartedModule
     public $kernel;
 
     /**
-     * @var \Symfony\Component\DependencyInjection\ContainerInterface
+     * @var ContainerInterface
      */
-    public $container;
+    private $container;
```

Comment:
Ok, you can do it to handle the BC on 2.1.x. We should remove this 'magic' in a future release, maybe 3.0, and have only `_getContainer`.